virttest/virt_vm.py

```diff
@@ -577,6 +577,9 @@ def needs_restart(self, name, params, basedir):
             return True
 
         try:
+            # If `params.get("spice*_port") == generate` and
+            # `keep_guest_running == True`, `need_restart` will
+            # always be True to prevent assigning of undesirable SPICE port.
```

Comment:
The `virt_vm` is generic implementation and this is a specific information about `qemu_vm` and only when using `spice` vga. Would you please move this note to the place where the `spice_*port` is being set? Also I'd suggest the `FIXME: ` prefix.

Reply:
Also please include that in commit message (and please be more verbose in the commit message)